Question5.java

```diff
@@ -0,0 +1,191 @@
+/**
+ * Stack interface
+ */
+interface Stack<Type>
+{
+    void push(Type data) throws StackSizeExceeded;
+    void pop();
```

Comment:
Pop and peek should return the data

Reply:
I have resolved this issue. now pop and peek returns the data.